sunpy/net/dataretriever/sources/goes.py

```diff
@@ -32,10 +32,10 @@ def _get_goes_sat_num(self, start, end):
 
         sat_list = []
         for sat_num in goes_operational:
-            if ((start > goes_operational[sat_num].start and
-                 start < goes_operational[sat_num].end) and
-                (end > goes_operational[sat_num].start and
-                 end < goes_operational[sat_num].end)):
+            if ((start >= goes_operational[sat_num].start and
```

Comment:
@DanRyanIrish Does this invalidate this code? i.e. were these dates meant to be exclusive or inclusive?